server/routes/checkAnswers.js

```diff
@@ -47,3 +150,64 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
 
   return router
 }
+
+// utility function - convert to string where input MAY be an array
+
+const convertNestedArraysToString = allElements => {
+  if (Array.isArray(allElements)) {
+    let finalString = ''
+    allElements.forEach(element => {
+      finalString += `${element}, `
+    })
+    finalString = finalString.slice(0, -2)
+    return finalString
+  }
+  return allElements
+}
+
+// utility function - convert to string when input WILL be nested objects
+const convertArrayOfObjectsToString = (dataArray = []) => {
+  let finalString = ''
+  dataArray.forEach(element => {
+    finalString += `${element.name}, `
+  })
+  return finalString.slice(0, -2)
+}
+
+// titlecase all person names
+const toTitleCase = (str = '') => {
+  return str.replace(/\w\S*/g, txt => {
+    return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase()
+  })
+}
+
+// staff taken to hospital
+const staffTakenToHospital = (staffMembers = []) => {
+  const hospitalisedStaff = staffMembers.filter(staff => staff.hospitalisation === 'Yes').map(staff => staff.name)
+  if (hospitalisedStaff.length === 0 && staffMembers.length > 0) {
+    return 'none'
+  }
+  return toTitleCase(hospitalisedStaff.join(', '))
+}
+
+// evidence bagged
+const baggedAndTaggedEvidence = (tagsAndEvidence = [], evidenceYesNo = '') => {
+  if (evidenceYesNo === 'No') {
+    return 'none'
+  }
+  return tagsAndEvidence
+    .map(item => {
+      return `${item.name} ${item.description}`
+    })
+    .join(`<br/>`)
+}
+
+// how many officers involved
```

Comment:
the comments on `howManyOfficersInvolved` and `typeOfHandcuffsUsed` seem a bit superfluous - the method name should give enough context

Reply:
Done. It's just a personal preference to add comments as I find it quicker to distinguish code sections on a page